.github/CODEOWNERS

```diff
@@ -1312,6 +1313,10 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /x-pack/test_serverless/**/test_suites/common/management/transforms/ @elastic/ml-ui
 
 # Additional plugins and packages maintained by the ML team.
+/x-pack/test/functional/apps/aiops @elastic/ml-ui
+/x-pack/test/api_integration_basic/apis/aiops @elastic/ml-ui
```

Comment:
Nit - could you move the `/x-pack/test/api_integration_basic` and `/x-pack/test/api_integration` items so that they are below the `accessibility` item with the other api test items in this block.

Reply:
Yes - aim to group items together by folder. So the block should look something like this I think.

Note looks like you can add this file in too
```
/x-pack/test/api_integration/services/transform.ts @elastic/ml-ui
```

```
# Additional plugins and packages maintained by the ML team.
/x-pack/test/accessibility/apps/group2/transform.ts  @elastic/ml-ui
/x-pack/test/api_integration/apis/aiops/ @elastic/ml-ui
/x-pack/test/api_integration/apis/transform/ @elastic/ml-ui
/x-pack/test/api_integration_basic/apis/aiops @elastic/ml-ui
/x-pack/test/api_integration_basic/apis/transform/ @elastic/ml-ui
/x-pack/test/api_integration/services/aiops.ts @elastic/ml-ui
/x-pack/test/api_integration/services/transform.ts @elastic/ml-ui
/x-pack/test/functional/apps/aiops @elastic/ml-ui
/x-pack/test/functional/apps/transform/ @elastic/ml-ui
/x-pack/test/functional/services/transform/ @elastic/ml-ui
/x-pack/test/functional_basic/apps/transform/ @elastic/ml-ui
```

Reply:
https://github.com/elastic/kibana/pull/197468/commits/d6c387df27dbebe99e56deff825dc6d3b534e925